File: day005/day005.cs
using System;

	class Day005 {
		static void Main() {
			int x, y, temp;
			Console.Write("Bir sayı giriniz: X ");
			x = Convert.ToInt32(Console.ReadLine());
			Console.Write("Başka bir sayı daha giriniz: Y ");
			y = Convert.ToInt32(Console.ReadLine());
			temp = x;
			x = y;
			y = temp;
			Console.Write("x:" + x + " y: " + y); 
		}
}

File: day005/day005.c
#include <stdio.h>

int main() {
	int x, y, temp; 
	printf("Bir sayı giriniz: ");
	scanf("%d", &x);
	printf("Başka bir sayı daha giriniz: ");
	scanf("%d", &y);
	temp = x;
	x = y;
	y = temp;
	printf("%d sayısı x, %d sayısı y", x, y);
}
